src/demo-app/a11y/menu/menu-a11y.html

```diff
@@ -0,0 +1,54 @@
+<div class="demo-menu">
+  
+  <section>
+    <h2>Click on the icon will trigger menu</h2>
+    <md-menu #menu1="mdMenu">
+      <button md-menu-item> Settings </button>
+      <button md-menu-item> Help </button>
+    </md-menu>
+
+    <button md-icon-button [mdMenuTriggerFor]="menu1">
+      <md-icon>more_vert</md-icon>
+    </button>
+  </section>
+
+  <section>
+    <h2>Display menu options with icon</h2>
+    <button md-icon-button [mdMenuTriggerFor]="menu2">
+      <md-icon>more_vert</md-icon>
+    </button>
+    <md-menu #menu2="mdMenu">
+      <button md-menu-item>
+        <md-icon>dialpad</md-icon>
+        <span>Redial</span>
+      </button>
+      <button md-menu-item>
+        <md-icon>voicemail</md-icon>
+        <span>Check voicemail</span>
+      </button>
+      <button md-menu-item>
+        <md-icon>notifications_off</md-icon>
+        <span>Disable alerts</span>
+      </button>
+    </md-menu>
+  </section>
+
+  <section>
+    <h2>Menu with links</h2>
+    <button md-icon-button [mdMenuTriggerFor]="menu2">
```

Comment:
The button needs a label that communicates "Here's some navigation links", maybe something like "Learn more about Angular" (with items that match)